src/stream/src/executor/hash_agg.rs

```diff
@@ -670,12 +650,26 @@ impl<K: HashKey, S: StateStore> HashAggExecutor<K, S> {
                 }
                 Message::Chunk(chunk) => {
                     Self::apply_chunk(&mut this, &mut vars, chunk).await?;
+
+                    if this.max_dirty_groups_heap_size > 0
```

Comment:
Does this mean that we use `0` to disable this feature? From my own perspective, `0` seems also to be a reasonable configuration. What about instead using an `Option` here?

Reply:
I do agree, but seems in TOML user can not specify NULL value? We can't use None when user doesn't set it because we want set a reasonable value to prevent from OOM by default.

Reply:
> We can't use None when user doesn't set it

Agree, but I guess the default value from `serde` will handle this for us.